guides/common/modules/proc_setting-minimal-data-collection.adoc

```diff
@@ -9,7 +9,7 @@ Reports are still processed daily during the midnight synchronization cycle.
 Minimal reports also exclude hostnames, IP addresses, and installed packages.
 Therefore, obfuscation and exclusion settings do not apply.
 
-If you switch to *Optional data collection*, {Project} enables the following settings by default:
+If you switch to *Analytics Data Collection*, {Project} enables the following settings by default:
```

Comment:
```suggestion
If you switch to *Analytics data collection*, {Project} enables the following settings by default:
```
You use lower case capitalization in the UI - https://github.com/theforeman/foreman_rh_cloud/pull/1007